grails-app/i18n/messages.properties

```diff
@@ -1456,6 +1457,7 @@ location.bgColor.label=Background color
 location.fgColor.label=Foreground color
 location.bgColor.invalid.matchingcolor=Colors cannot be the same
 location.supportedActivities.invalid.supportedActivities=You cannot set supported activity NONE in combination with other supported activities
+location.supportedActivities.noRequestApprovers=You must add one or several Approvers to the location to support the requests approval workflow. Go to users list to add an Approver.
```

Comment:
unfortunately it was not possible to make the link to users list work fine here, since we render the errors on location edit page like that:
```gsp
<g:hasErrors bean="${locationInstance}">
        <div class="errors">
            <g:renderErrors bean="${locationInstance}" as="list"/>
        </div>
</g:hasErrors>
```
When added a href tag to the message, it was reading it as a string:
![Screenshot from 2023-09-13 09-34-56](https://github.com/openboxes/openboxes/assets/93163821/6fb6ba8d-5412-4024-be6d-5e32dc68582d)